sources/core/Stride.Core/ThreadThrottler.cs

```diff
@@ -8,12 +8,14 @@ namespace Stride.Core
     public class ThreadThrottler
     {
         /// <summary>
-        /// Minimum time allowed between each call
+        /// Will block until this amount of time passed. 
```

Comment:
As a personal preference, I would have kept the original description in the summary, and put all those additional lines under a `<remarks>` section.

The summary is what Visual Studio shows by default in a tooltip when hovering the mouse over it. So I like to keep it as short as possible. The remarks and other optional elements are usually shown only when displaying the full information.

I might be biased against the behavior that resharper shows since it can display that full information when pressing CTRL+SHIFT+F1. Visual Studio currently doesn't have the capacility but it is discussed in https://github.com/dotnet/roslyn/issues/31618.

Reply:
Issue is that people using this API more often than not want to throttle based on the amount of frames per second not the amount of seconds per frame, it most likely is hard for them to wrap their head around this so that comment is kind of essential.
We could also not require user to do that extra step in the first place and there's two way to go about this:
* Present users with a property that takes and shows that data in a 'fps' format.
This will lead to precision issues as converting between the two will be float-based and won't be obvious to the user which might in term lead to other user issues like running logic doing equality comparison with this property.
* Changing the entire system to use an fps format, doing so will make the system less precise than it is right now when users expect this to be quite precise in the first place.
Obviously those are fairly nitpick-y but I don't feel right loosing any kind of precision to the system to work around this specific issue.

Given what I said, do you still feel that this should go within remarks ?

Reply:
> set this value to 10

To 10 what? milliseconds, seconds, minutes, ticks?

I think, according to the previous comment on `GameBase.WindowMinimumUpdateRate` (now removed) that it is supposed to be `TimeSpan.FromSeconds(1.0/10)`. We should make that clear in the comment.

OK to keep it in the summary for now.

Reply:
Ok for me after commit 25b1b56.

Reply:
No reason for 10, probably mistyped that one while pushing to this pr.